environments/staging/inventory.ini.j2

```diff
@@ -105,12 +105,12 @@ redis2
 
 [elasticsearch:children]
 # Amazon EC2
-# old ES v2 hosts
-# es4
-# es5
-# ES v7 hosts
-es6
-es7
+# ES v7.9.1 hosts
+es4
+es5
```

Comment:
Did you reuse the old ES2 VMs, or are `es4` and `es5` new VMs? It's probably too late, but it's not a good idea to reuse host names because that can make it difficult to track the history any place where those names are referenced. Better to retire old names and make new ones for new things.

Reply:
Hey, I reused the same machines, haven't created new ones, so hostnames shouldn't be an issue